### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by zepfietje**

### CODE REVIEW

File: packages/tables/resources/lang/en/table.php [hunk 1]
```diff
@@ -16,17 +16,17 @@
...
     'fields' => [
 
-        'search_query' => [
-            'label' => 'Search',
-            'placeholder' => 'Search',
-        ],
-
         'bulk_select_page' => [
-            'label' => 'Select / Deselect all items for bulk actions.',
+            'label' => 'Select/Deselect all items for bulk actions.',
```
Comment (danharrin) [inline packages/tables/resources/lang/en/table.php]:
```suggestion
            'label' => 'Select/deselect all items for bulk actions.',
```

Comment (zepfietje) [reply packages/tables/resources/lang/en/table.php]: The space should not be there.

File: packages/tables/resources/lang/en/table.php [hunk 2]
```diff
@@ -16,17 +16,17 @@
...
         ],
 
         'bulk_select_record' => [
-            'label' => 'Select / Deselect item :key for bulk actions.',
+            'label' => 'Select/Deselect item :key for bulk actions.',
```
Comment (zepfietje) [inline packages/tables/resources/lang/en/table.php]:
```suggestion
            'label' => 'Select/deselect item :key for bulk actions.',
```

File: packages/tables/resources/lang/en/table.php (hunk 1 above)
Comment (danharrin) [reply packages/tables/resources/lang/en/table.php]: hmm why?

Comment (zepfietje) [reply packages/tables/resources/lang/en/table.php]: https://www.grammarly.com/blog/slash/

"When a slash signifies alternatives between only two words, don’t use spaces before or after."

**Pull request merged**